lumicks/pylake/population/mixture.py

```diff
@@ -161,12 +161,12 @@ def extract_dwell_times(self, trace, *, exclude_ambiguous_dwells=True):
         return dwell_times
 
     @property
-    def bic(self):
+    def bic(self) -> float:
         """Bayesian Information Criterion."""
```

Comment:
Might be nice to expand the docstring for these information criteria a little bit if we're touching this up anyway?